deploy/sample-backup-configs/sample_config_runtime.ini

```diff
@@ -57,3 +57,8 @@ suffixtag=
 # In general, this is not required to produce deterministic simulations on
 # target machines running linux. Enable if you observe simulation non-determinism.
 zerooutdram=no
+
+[synthprint]
```

Comment:
This probably needs a comment explaining what a base cycle is, or a link to the docs.